mscore/palette.h

```diff
@@ -95,6 +95,7 @@ class PaletteScrollArea : public QScrollArea {
       PaletteScrollArea(Palette* w, QWidget* parent = 0);
       bool restrictHeight() const { return _restrictHeight; }
       void setRestrictHeight(bool val) { _restrictHeight = val; }
+      virtual void keyPressEvent(QKeyEvent* event) override;
```

Comment:
Sure.  And then I guess figuring out how to get screenreader feedback for all of this navigation, otherwise it's unfortunately not of as much use to blind users as we'd hope.

But I do hope there is also time to finish the palette shortcut code, because it too is pretty close, and I think will make a lot of people (myself included!) very happy.